Library/Homebrew/extend/os/mac/utils/curl.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  module Curl
+    undef return_value_for_empty_http_status_code
+
+    def return_value_for_empty_http_status_code(url_type, url)
+      # Hack around https://github.com/Homebrew/brew/issues/3199
+      return if MacOS.version == :el_capitan
+
+      generic_return_value_for_empty_http_status_code url_type, url
```

Comment:
I reckon this can go since we don't allow system curl on El Capitan anymore.

Reply:
Good catch @Bo98!